content-src/lib/shim.js

```diff
@@ -1,7 +1,48 @@
 const fakeFrecent = [
   {
     title: "Facebook",
-    leadImage: "https://www.facebook.com/images/fb_icon_325x325.png",
```

Comment:
was `leadImage` a typo? I have `url(${site.leadImage || site.image})` in the TopSites ^^ component. I can remove if not needed.